### PR TITLE
Fix up links to bodhi and koji.

### LINKED ISSUE #93: Builds list does not link to koji
I'm on https://apps.fedoraproject.org/packages/moin/builds/
and particular items in the build list are links, which should (imho) link to appropriate koji builds.
However, all links link to https://apps.fedoraproject.org/packages/package_maintenance/tools/builds?package=moin

### LINKED ISSUE #146: package versions should link to koji build
Some package versions listed in "packages" link to a bodhi update page while others aren't links at all. I think it would be a great improvement if all versions linked consistently to koji builds and those with updates had an additional link to bodhi.

It might also be useful to show which version of a package has been released originally with the respective Fedora release, as that information would give one a hint of a range of versions that can bee seen in an installation of that release of Fedora whether updated or not.

### PR DESCRIPTION
**Pull request by ralphbean**
- Fixes #146.
- Fixes #93.

### CODE REVIEW

File: fedoracommunity/connectors/bodhiconnector.py [hunk 1]
```diff
@@ -579,21 +585,25 @@ def query_active_releases(self, filters=None, **params):
                     if query.endswith('-testing'):
                         nvr = parse_build(release['nvr'])
                         row['testing_version'] = HTML.tag(
-                            'a', c='%(version)s-%(release)s' % nvr,
-                            href='%s/updates/%s' % (self._prod_url, nvr['nvr']))
+                            'a',
+                            c='%(version)s-%(release)s' % nvr,
+                            href=koji_build_url % nvr)
```
Comment (pypingou) [inline fedoracommunity/connectors/bodhiconnector.py]: for updates we point to koji and for updates-testing to bodhi?

Comment (ralphbean) [inline fedoracommunity/connectors/bodhiconnector.py]: - for all the build nvrs, point to koji.
- if there is also an associated update, then add a `(update)` link which points to bodhi.

Comment (pypingou) [inline fedoracommunity/connectors/bodhiconnector.py]: But here, aren't we in the ``updates-testing`` update?

Comment (ralphbean) [inline fedoracommunity/connectors/bodhiconnector.py]: We are.  But like I said, this links to the koji build directly for all build nvrs.
And then additionally, later, an extra link is tacked on to add a link to the bodhi update.

**Comment by pypingou** [comment]
After some more discussion on IRC :+1: for this

**Comment by ralphbean** [comment]
Thanks for wading through that with me!

**Pull request merged**